website/docs/sql-reference/statements/sql-select.md

```diff
@@ -0,0 +1,49 @@
+---
+id: select
+title: SQL SELECT
+description: The SQL SELECT statement is the fundamental building block of any query that allows you to select specific columns from a database schema object.
+slug: /sql-reference/select
+---
+
+<head>
+    <title>Working with SQL SELECT statements</title>
+</head>
+
+My goodness, would there even be modern data teams without SQL SELECT statements? Probably not.
+
+Luckily, we live in a world of tabular data, cloud data warehouses, and SQL prowess. Analysts and analytics engineers are writing queries, creating data models, and leveraging SQL to power their data transformations and analysis. But what makes these queries possible? SELECT statements.
+
+The SQL SELECT statement is the fundamental building block of any query: it allows you to select specific columns (data) from a database schema object (table/view). In a dbt project, a SQL dbt model is technically a singular SELECT statement (often built leveraging CTEs or subqueries).
+
+## How to use SELECT
+
+Any query begins with a simple SELECT statement:
+
+```sql
+select
+	order_id, --your first column you want selected
+	customer_id, --your second column you want selected
+	order_date --your last column you want selected (and so on)
+from {{ ref('orders') }} --the table/view/model you want to select from
+limit 3
+```
+
+This basic query is selecting three columns from the [jaffle shop’s](https://github.com/dbt-labs/jaffle_shop/blob/main/models/orders.sql) `orders` table and returning three rows. If you execute this query in your data warehouse, it will return a result looking like this:
```

Comment:
FYI noticing jaffle shop is capitalized elsewhere but not here

Reply:
Great catch!